packages/docs-app/src/examples/core-examples/drawerExample.tsx

```diff
@@ -0,0 +1,167 @@
+                                engineers — and leaders in their organization’s data revolution.
+                            </p>
+                            <p>
+                                Foundry’s back end includes a suite of best-in-class data integration capabilities: data
+                                provenance, git-style versioning semantics, granular access controls, branching,
+                                transformation authoring, and more. But these powers are not limited to the back-end IT
+                                shop.
+                            </p>
+                            <p>
+                                In Foundry, tables, applications, reports, presentations, and spreadsheets operate as
+                                data integrations in their own right. Access controls, transformation logic, and data
+                                quality flow from original data source to intermediate analysis to presentation in real
+                                time. Every end product created in Foundry becomes a new data source that other users
+                                can build upon. And the enterprise data foundation goes where the business drives it.
+                            </p>
+                            <p>
+                                Palantir Foundry radically reimagines the way enterprises interact with data by
+                                amplifying and extending the power of data integration. With Foundry, anyone can source,
+                                fuse, and transform data into any shape they desire. Business analysts become data
+                                engineers — and leaders in their organization’s data revolution.
+                            </p>
+                            <p>
+                                Foundry’s back end includes a suite of best-in-class data integration capabilities: data
+                                provenance, git-style versioning semantics, granular access controls, branching,
+                                transformation authoring, and more. But these powers are not limited to the back-end IT
+                                shop.
+                            </p>
+                            <p>
+                                In Foundry, tables, applications, reports, presentations, and spreadsheets operate as
+                                data integrations in their own right. Access controls, transformation logic, and data
+                                quality flow from original data source to intermediate analysis to presentation in real
+                                time. Every end product created in Foundry becomes a new data source that other users
+                                can build upon. And the enterprise data foundation goes where the business drives it.
+                            </p>
```

Comment:
this is far too much content! maybe set a max-width of like 700 on a centered container inside the body?

Reply:
ah my fault. I doubled the content to make it scrollable